plugins/cue_points/quiz/QuizPlugin.php

```diff
@@ -920,7 +920,7 @@ private static function getQuizElasticSearchData($entry)
 		$isQuiz = self::getQuizData($entry);
 		if (!is_null($isQuiz))
 		{
-			$quizData['is_quiz'] = true;
+			$quizData['is_quiz'] = Kaltura_Client_Enum_NullableBoolean::TRUE_VALUE;
```

Comment:
You should use: KalturaNullableBoolean::TRUE_VALUE

Reply:
yep my bad